packages/media-module/src/components/Main/ThumbnailView.module.css

```diff
@@ -2,4 +2,5 @@
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
     gap: var(--theme-spacing-Full);
+    margin-bottom: 4rem;
```

Comment:
I think the correct place solution is adding `padding-bottom: 41px;`  when in `selectionMode`. As the calculation for the height doesn't correctly measure the available size.